packages/utils/odsp-doclib-utils/src/test/tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "rootDir": "./",
         "outDir": "../../dist/test",
         "types": [
+            "node",
```

Comment:
Is this addition intentional?  Looks like you're adding node types in 6 packages in this change.

Reply:
Ah I see it's because they're all test configs, never mind.

Reply:
Yes, these are all configs for tests only (separate from main packages), and they are only ran in node.